crates/ruff_python_formatter/src/comments/placement.rs

```diff
@@ -1805,18 +1805,6 @@ fn handle_lambda_comment<'a>(
     locator: &Locator,
 ) -> CommentPlacement<'a> {
     if let Some(parameters) = lambda.parameters.as_deref() {
-        // Comments between the `lambda` and the parameters are dangling on the lambda:
```

Comment:
We should revert this change but it triggers an instability that I haven't yet been able to figure out